third_party/move/move-compiler-v2/tests/testsuite.rs

```diff
@@ -580,6 +580,24 @@ const TEST_CONFIGS: Lazy<BTreeMap<&str, TestConfig>> = Lazy::new(|| {
             stop_after: StopAfter::SecondAstPipeline,
             ..config().exp(Experiment::MESSAGE_FORMAT_JSON)
         },
+        TestConfig {
+            name: "cmp-rewrite-lang-v2.1",
+            runner: |p| run_test(p, get_config_by_name("cmp-rewrite-lang-v2.1")),
+            include: vec!["/cmp-rewrite/"],
+            exp_suffix: Some("v2.1.exp"),
+            stop_after: StopAfter::SecondAstPipeline,
```

Comment:
We should not use `stop_after`, so that we can check bytecode verification result. Same for below. We may even want to display the final generated file format bytecode as this is an optimization.